src/library_stack.js

```diff
@@ -5,15 +5,22 @@
  */
 
 mergeInto(LibraryManager.library, {
-#if WASM_BACKEND == 0
+  emscripten_stack_get_base: function() {
+    return STACK_BASE;
+  },
+  emscripten_stack_get_end: function() {
+    return STACK_MAX;
```

Comment:
if we want to move from base/top (or base/max) to base/end, how about a TODO to rename `STACK_MAX` etc. too?

(base/end sounds reasonable to me.)

Reply:
SGTM.